mne_bids/tests/test_utils.py

```diff
@@ -36,3 +41,50 @@ def test_make_folders():
     output_path = _TempDir()
     make_bids_folders(subject='hi', kind='ba', root=output_path)
     assert os.path.isdir(os.path.join(output_path, 'sub-hi', 'ba'))
+
+
+def test__check_types():
+    """Test the check whether vars are str or None."""
+    assert _check_types(['foo', 'bar', None]) is None
+    with pytest.raises(ValueError):
+            _check_types([None, 1, 3.14, 'eeg', [1, 2]])
+
+
+def test_brainvision_utils():
+    """Test generation of brainvision data and moving it around."""
+    # Make some test brainvision data
+    bv_ext = ['.eeg', '.vhdr', '.vmrk']
+    basename = 'testnow'
+    data_dir = _TempDir()
+    _vhdr = make_test_brainvision_data(output_dir=data_dir, basename=basename)
+
+    # Assert that we can read it
+    raw = read_raw_brainvision(_vhdr)
+    assert isinstance(raw, BaseRaw)
+
+    bv_file_paths = []
+    for ext in bv_ext:
+        bv_file_paths.append(os.path.join(data_dir, basename + ext))
+
+    # quick check of make_test_brainvision_data's return value
+    assert _vhdr in bv_file_paths
+
+    # Now try to move it to a new place and name
+    new_data_dir = _TempDir()
+    new_basename = 'testedalready'
+    new_bv_file_paths = []
+    for ext in bv_ext:
+        new_bv_file_paths.append(os.path.join(new_data_dir,
+                                              new_basename + ext))
+    for src, dest in zip(bv_file_paths, new_bv_file_paths):
+        copyfile_brainvision(src, dest)
+
+    # Assert we can move the new file
+    raw = read_raw_brainvision(os.path.join(new_data_dir,
+                                            new_basename + '.vhdr'))
+    assert isinstance(raw, BaseRaw)
+
+    # Assert that errors are raised
+    with pytest.raises(ValueError):
+        copyfile_brainvision('bogus.vmrk', 'bogus.vhdr')
```

Comment:
can you also test that MNE is able to read in the copied files correctly? after the pointers have changed ...

Reply:
see line 83, there is a typo in the comment though. It should read `# Assert we can read the new file`